lib/widgets/channel_colors.dart

```diff
@@ -41,26 +41,26 @@ abstract class ChannelColorSwatches {
     // so this is an important optimization.
     if (identical(a, b)) return a;
 
-    return _StreamColorSwatchesLerped(a, b, t);
+    return _ChannelColorSwatchesLerped(a, b, t);
   }
 }
 
-class _StreamColorSwatchesLight extends ChannelColorSwatches {
-  _StreamColorSwatchesLight();
+class _ChannelColorSwatchesLight extends ChannelColorSwatches {
```

Comment:
I am not sure, but I think this and all other subclasses of `ChannelColorSwatches` should also go in [`bdf6fe4`](bdf6fe406fcaf7404c360b7370ae65b0b399a2ef): channel_colors [nfc]: Rename StreamColorSwatches to ChannelColorSwatches.